examples/inferencing/running-on-gke.md

```diff
@@ -0,0 +1,185 @@
+   </td>
+   <td><strong>Example Value</strong>
+   </td>
+  </tr>
+  <tr>
+   <td><code>GPU_COUNT_PER_MODEL</code>
+   </td>
+   <td>Y
+   </td>
+   <td>Number of GPUs exposed to the pod, also used to set the parallelism when using FasterTransformer
+   </td>
+   <td><code>4</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODEL_SOURCE_PATH</code>
+   </td>
+   <td>Y
+   </td>
+   <td>GCS path pointing to the directory of the model to deploy.
+<p>
+Note: For a model fine tuned using the pipeline, look at the Model Artifact after the training step and use the URL property.
+   </td>
+   <td><code>gs://my-bucket/pipeline_runs/237939871711/llm-pipeline-20230328153111/train_5373485673388965888/Model/</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>EXISTING_CLUSTER_ID</code>
+   </td>
+   <td>N
+   </td>
+   <td>Name of an existing cluster (in the corresponding Region and Project) to use instead of provisioning a new cluster.
+   </td>
+   <td><code>my-gke</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>KSA_NAME</code>
+   </td>
+   <td>N
+   </td>
+   <td>Name of the Kubernetes Service Account configured with access to the given GCS path. By default one will be provisioned as ‘aiinfra-gke-sa’
+   </td>
+   <td><code>my-other-ksa</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODEL_NAME</code>
+   </td>
+   <td>N
+   </td>
+   <td>Friendly name for the model, used in constructing the Kubernetes Resource names
+   </td>
+   <td><code>t5-flan</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>INFERENCING_IMAGE_TAG</code>
+   </td>
+   <td>N
+   </td>
+   <td>Image tag for the inference image. Default is ‘release’
+   </td>
+   <td><code>latest</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>USE_FASTER_TRANSFORMER</code>
+   </td>
+   <td>N
+   </td>
+   <td>Boolean to set when the FasterTransformer / Triton path should be enabled.
+<p>
+This controls whether a Conversion job is scheduled, and the inference image that will be deployed.
+   </td>
+   <td><code>true</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>CONVERTED_MODEL_UPLOAD_PATH</code>
+   </td>
+   <td>Y*
+   </td>
+   <td>Only required when USE_FASTER_TRANSFORMER is set.
+<p>
+A GCS path to upload the model after it is converted for FasterTransformer
+   </td>
+   <td><code>gs://pirillo-sct-bucket/converted_t5/1/Model</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>POD_MEMORY_LIMIT</code>
+   </td>
+   <td>N
+   </td>
+   <td>Sets the memory limit of pods for GKE in <a href="https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory">Kubernetes Memory resource format</a>. Defaults to “16Gi”.
+   </td>
+   <td><code>50Gi</code>
+   </td>
+  </tr>
+</table>
+
+
+
+### Running the image
+
+The Cluster Provisioning + Deployment image is available at [gcr.io/llm-containers/gke-provision-deploy](gcr.io/llm-containers/gke-provision-deploy) .
+
+Run the image using the `docker run` command specifying the Environment File with `--env-file`.
+
```

Comment:
Please add a command that the user can copy and paste. The sample command should use the sample env file.